pep-0710.rst

```diff
@@ -48,16 +48,56 @@ give more accurate reports. Yet another use case could be reconstruction of the
 Python environment by pinning each installed package to a specific distribution
 artifact consumed from a Python package index.
 
-The motivation described in this PEP is an extension of those in :pep:`610`.  Besides
-stating information about packages installed using a direct URL, installers SHOULD
-record information also for packages installed from Python package indexes when
-identified by their name, and optionally their version.
-
 Rationale
 =========
 
-[describing the reasoning behind the approach chosen in this PEP, and/or a brief history of the discussions that led to this PEP being proposed]
-
+The motivation described in this PEP is an extension of those in :pep:`610`.
+Besides stating information about packages installed using a direct URL,
+installers should record information also for packages installed from Python
+package indexes when identified by their name, and optionally their version.
+
+The idea described in this PEP originated in a tool called `micropipenv
+<https://github.com/thoth-station/micropipenv>`__ that is used to install
+:term:`distribution packages <Distribution Package>` in containerized
+environments (see the reported issue `thoth-station/micropipenv#206`_). When
+installing a :term:`Distribution Package` during a containerized application
+build, the assembled containerized application does not implicitly carry
+information about the provenance of installed :term:`distribution packages
+<Distribution Package>` when installed using their name, and optionally their
+version.  This creates a requirement for container image suppliers to link
+container images with the corresponding build process, its configuration and
+the application source code for checking requirements files in cases when
+software present in containerized environments needs to be audited.
+
+The `subsequent discussion in the Discourse thread
+<pip_installation_report>`_ mentioned also
+a new option ``--report`` in pip that can generate a detailed JSON report about
```

Comment:
```suggestion
pip's new ``--report`` option that can
`generate a detailed JSON report <pip_installation_report_>`__ about
```

* Link the installation report here instead, where it is actually referenced in the text
* Use more idiomatic and readible word order